src/Components/pages/Books.js

```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import Book from './Book';
+import Form from './Form';
+
+const booklist = [
+  {
+    id: '01G',
+    title: 'The Hunger Games',
+    author: 'Suzanne Collins',
+  },
+  {
+    id: '02G',
+    title: 'The Hames',
+    author: 'Suz lins',
+  },
+  {
+    id: '03G',
+    title: 'Thames',
+    author: 'zanne Col',
+  }];
+
+const Books = () => (
+  <>
+    <div>
+      {
+        booklist.map((book) => (<Book key={book.id} title={book.title} author={book.author} />))
+      }
+    </div>
```

Comment:
- [ ] Since this is a "List" of books, it would be wise to use the `<ul>` tag instead of `<div>`. Remember that `<div>` tags don't provide any semantic meaning to the browser, so please consider using accessible tags whenever you can! 🤓 🔧 

```suggestion
    <ul>
      {
        booklist.map((book) => (<Book key={book.id} title={book.title} author={book.author} />))
      }
    </ul>
```